iroha/src/asset.rs

```diff
@@ -525,6 +525,63 @@ pub mod query {
             }))
         }
     }
+
+    /// To get the state of all assets in an account filtered by assets definition,
+    /// GetAccountAssetsWithDefinition query can be used.
+    #[derive(Clone, Debug, Io, IntoQuery, Encode, Decode)]
+    pub struct GetAccountAssetsWithDefinition {
```

Comment:
I think it is better to add new queries in a separate PR.

Reply:
But tests will not work without them

Reply:
Ah, I see.